### PR TITLE
Add Cloud Monitoring and Test Engineer position.

### PR DESCRIPTION
**Pull request by garloff**

### CODE REVIEW

File: _i18n/de/jobs/monitoringtest-engineer.md [hunk 1]
```diff
@@ -112,10 +76,10 @@ Wir bieten
   den relevanten Open Source Communities weltweit zusammenarbeiten können,
 * die Möglichkeit, zu einem Projekt beizutragen, das maßgeblich für ein
   lebendiges Ökosystem von föderierten Infrastrukturanbietern in Europa und
-  idarüber hinaus sein will,
+  darüber hinaus sein will,
 * eine Schlüsselrolle in diesem Projekt von herausgehobener Bedeutung,
 * die Mitarbeit an hochmoderner Open-Source-Infrastrukturtechnologie,
-* viel Flexibilität, wie Du Deine Arbeit organisieren und zu dem international
+* viel Flexibilität, wie Sie Ihre Arbeit organisieren und zu dem international
```
Comment (fkr) [inline _i18n/de/jobs/monitoringtest-engineer.md:82]: Imho I'd stay with the "Du" - this is by now very typical and for the way we work much nicer.

**Review by fkr — APPROVED** [review]
Thanks @alexander-diab - I really like your changes. I left one comment (regarding the change from 'Du' to 'Sie').

File: _i18n/de/jobs/monitoringtest-engineer.md (hunk 1 above)
Comment (garloff) [reply _i18n/de/jobs/monitoringtest-engineer.md:82]: We just need to be consistent within the job ad or - better - across all of them.

Comment (fkr) [reply _i18n/de/jobs/monitoringtest-engineer.md:82]: @garloff currently we have one (https://scs.community/de/jobs/po-ops) that makes use of 'Sie' and one (https://scs.community/de/jobs/product-engineer) that makes use of 'Du'.

Comment (garloff) [reply _i18n/de/jobs/monitoringtest-engineer.md:82]: Let's push this one out.
I'd consider making the product-engineer and this position use the "Du" as a second step.

**Pull request merged**